scheduler/compileserver.cpp

```diff
@@ -152,6 +160,7 @@ bool CompileServer::is_eligible(const Job *job)
            && (m_chrootPossible || job->submitter() == this)
            && load_okay
            && version_okay
+           && m_acceptingInConnection
```

Comment:
Instead of using a separate list from css (css_temp_hold) for CompileServers that are not accepting incoming connections, the CompileServer member function is_eligible() will return false.